domains/akhileshh.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Akhileshrangani4",
+        "email": "akhileshrangani4@gmail.com"
+    },
+    "record": {
+        "TXT": "google-site-verification=mxvu1et47munvjqueyoocxtlg8u1ikizyueuxntqtka"
```

Comment:
Google site verification through TXT records is not supported, use the HTTPS method